CE: Mention that you can use ValueChangeMode for eagerness (#137)
vaadin/docs: articles/ce/collaboration-binder.asciidoc

generic type for deserializing the value.
Note that if the element type is not supported by Collaboration Engine (e.g. `CheckboxGroup<Person>`),
you need to implement custom (de)serializer functions anyway.

> == Propagate Values Eagerly in Text Fields

Reviewer: ```suggestion
pass:[<!-- vale Vaadin.Weasel = NO -->]

== Propagate Values Eagerly in Text Fields
```

On text fields, the default and recommended mode for propagating values from one user to others are when the user blurs the field. 
However, in some use cases you might want to send the value of the field more eagerly. 

> You can configure how eagerly the data should be sent through the text fields own API, `TextField.setValueChangeMode(ValueChangeMode);`.

Reviewer: ```suggestion
You can configure how eagerly Text Field sends data through its own API, using `TextField.setValueChangeMode(ValueChangeMode);`.

pass:[<!-- vale Vaadin.Weasel = YES -->]
```

generic type for deserializing the value.
Note that if the element type is not supported by Collaboration Engine (e.g. `CheckboxGroup<Person>`),
you need to implement custom (de)serializer functions anyway.

pass:[<!-- vale Vaadin.Weasel = NO -->]

== Propagate Values Eagerly in Text Fields

> On text fields, the default and recommended mode for propagating values from one user to others are when the user blurs the field. 

Reviewer 2: ```suggestion
On text fields, the default and recommended mode for propagating values from one user to others is when the user blurs the field or presses the enter key.
```

> However, in some use cases you might want to send the value of the field more eagerly. 
> You can configure how eagerly Text Field sends data through its own API, using `TextField.setValueChangeMode(ValueChangeMode);`.

Reviewer 2: The first sentence seems redundant.
The API exists on most fields, so I would be more generic here and leave `TextField` to the example code.
```suggestion
You can configure how eagerly the field sends data through its own API, using the `setValueChangeMode(ValueChangeMode)` method.
```

pass:[<!-- vale Vaadin.Weasel = YES -->]
For example, to instantly send each keystroke to other users, you would do the following:

[source, java]
----

> textField.setValueChangeMode(ValueChangeMode.EAGER);

Reviewer 2: ```suggestion
TextField textField = new TextField();
textField.setValueChangeMode(ValueChangeMode.EAGER);
```

----

> Modes like `ValueChangeMode.LAZY` and `ValueChangeMode.TIMEOUT` can also be used together with `textField.setValueChangeTimeout(int);` to reduce the amount of traffic.

Reviewer 2: ```suggestion
Modes like `ValueChangeMode.LAZY` and `ValueChangeMode.TIMEOUT` can also be used together with the `setValueChangeTimeout(int)` method to reduce the amount of traffic.
```
